os/tools/build_test.sh

```diff
@@ -55,3 +55,12 @@ for target in ${build_targets[@]}; do
 	echo "======== \"${target}\" Built ========"
 	echo "===================================="	
 done
+
+	echo "===================================="
+	echo "======== Clean build output ========"
+	echo "===================================="
+	cd ${OSDIR}; ${CLEAN_CMD}
+
+	echo "===================================="
+	echo "====== Build Test All Success ======"
+	echo "===================================="
```

Comment:
1. This is not a part of "adding build test option in dbuild". Let's split commits.
2.  The dbuild script prints start of build test, but this script does end. It looks strange.